sign-and-combine-ecdsa/nodejs/src/index.ts

```diff
@@ -0,0 +1,186 @@
+import * as ethers from "ethers";
+import { LitContracts } from "@lit-protocol/contracts-sdk";
+import { LitNetwork, LIT_RPC, LIT_CHAINS } from "@lit-protocol/constants";
+import { LitNodeClient } from "@lit-protocol/lit-node-client";
+import {
+  LitAbility,
+  LitActionResource,
+  LitPKPResource,
+  createSiweMessageWithRecaps,
+  generateAuthSig,
+} from "@lit-protocol/auth-helpers";
+
+import { getEnv } from "./utils";
+import { litActionCode } from "./litAction";
+
+const ETHEREUM_PRIVATE_KEY = getEnv("ETHEREUM_PRIVATE_KEY");
+let capacityTokenId = getEnv("CAPACITY_TOKEN_ID");
+let LIT_PKP_PUBLIC_KEY = getEnv("LIT_PKP_PUBLIC_KEY");
+let LIT_PKP_ETH_ADDRESS = getEnv("LIT_PKP_ETH_ADDRESS");
+
+const chain = "sepolia";
+const rpcUrl = LIT_CHAINS[chain].rpcUrls[0];
+const chainId = LIT_CHAINS[chain].chainId;
```

Comment:
`chain` should be replaced by a `const` that set to the ENV I mentioned above. We should write a helper function called `getChainInfo` the attempts to get `rpcUrl` and `chainId`, but if `chain` isn't supported, then we throw an error saying so